src/suresh/maxLength.java

```diff
@@ -0,0 +1,41 @@
+package suresh;
+
+public class maxLength {
+
+	String[] arr = { "Harsh,", "Maulik", "Technocredits", "krisna" };
+
+	static String getmaxLength(String[] arr) {
+		int maxLength = 0;
+		String longestString = null;
+
+		for (String s : arr)
+
+			if (s.length() > maxLength) {
+				maxLength = s.length();
+				longestString = s;
+			}
+		return longestString;
+	}
+
+	public static void main(String[] args) {
+
+		maxLength maxlenth = new maxLength();
+		String[] arr = { "Harsh,", "Maulik", "Technocredits", "krisna" };
+
+		String longestString = getmaxLength(arr);
+
+		System.out.format("longest string: '%s'\n", longestString);
```

Comment:
actually sop looks good here. format method should be used when you want to print multiple variable values.  how format method can help us here.